src/main/java/edu/hm/hafner/analysis/registry/VeracodePipelineScannerDescriptor.java

```diff
@@ -0,0 +1,34 @@
+package edu.hm.hafner.analysis.registry;
+
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.parser.VeracodePipelineScannerParser;
+
+import static j2html.TagCreator.*;
+
+/**
+ * A descriptor for Veracode Pipeline Scanner.
+ *
+ * @author Juri Duval
+ */
+class VeracodePipelineScannerDescriptor extends ParserDescriptor {
+    private static final String ID = "veracode-pipeline-scanner";
```

Comment:
Since this ID is used as URL, wouldn't "veracode" be more appropriate? Or is this URL already used by the Veracode Jenkins plugin?

Reply:
I was thinking of naming it `veracode` but Veracode offers a few other cli tools like [vosp-api-wrapper-java](https://docs.veracode.com/r/t_working_with_java_wrapper) and [srcclr](https://docs.veracode.com/r/Using_the_Veracode_SCA_Command_Line_Agent), so I wanted to give more explicit naming.

For example:
`veracode-srcclr`
`veracode-vosp-api-wrapper`
`veracode-pipeline-scanner`

Reply:
Ok, then let's stick with `veracode-pipeline-scanner`